### PR TITLE
fix: bug when inc or dec is used and kv does not exist it should add …

### PR DESCRIPTION
**Pull request by twilson63**
…0 and then do its task #2

### CODE REVIEW

File: deno/mod.ts [hunk 1]
```diff
@@ -17,29 +27,39 @@ interface Hyper {
   };
 }
 
+const exists = (c: Count): Count =>
+  ifElse(
+    (c: Count): boolean => compose(is(Number), prop("count"))(c),
```
Comment (TillaTheHun0) [inline deno/mod.ts:32]: If `count` is mistakenly cast to a non-number, ie the string `"5"`, then that count will be lost and reset to 0. 

Should this util simply check for existence of `count` and then attempt cast if it exists? Or is that out of this utils purview?

**Review by TillaTheHun0 — APPROVED** [review]
Just had a question. Otherwise it looks good to me.

File: deno/mod.ts (hunk 1 above)
Comment (twilson63) [reply deno/mod.ts:32]: It should always be a number

Comment (twilson63) [reply deno/mod.ts:32]: I am going to leave it as implemented

**Pull request merged**